CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
-## Unreleased
+# Changelog
 
-- Fixes a TS compilation error due to a broken entry point path.
-- Re-exports `NetworkData` type to be imported from `instabug-reactnative`.
+## [11.9.1](https://github.com/Instabug/Instabug-React-Native/compare/v11.9.0...HEAD) (March 01, 2023)
```

Comment:
**suggestion:** This URL works for "Unreleased" Section and not for Versions.

```suggestion
## [11.9.1](https://github.com/Instabug/Instabug-React-Native/compare/v11.9.0...v11.9.1) (March 01, 2023)
```

Reply:
Nice catch! 👌